compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/ComposeUIViewControllerConfiguration.uikit.kt

```diff
@@ -35,6 +35,11 @@ class ComposeUIViewControllerConfiguration {
 
     @ExperimentalComposeApi
     var platformLayers: Boolean = false
+
+    /**
+     * Determines whether the Compose view should have an opaque background.
+     */
+    var opaque: Boolean = true
```

Comment:
Please mark it as `@ExperimentalComposeApi` for the first release

Reply:
Dure, done!